### DISCUSSION TITLE
Consulta de como usar la API de MercadoLibre

### DISCUSSION THREAD

**Discussion by TomasM99** [dds-jv/foro#139] [TPA - 2020] [answered]
Buenas, tenemos dudas de como se usa el cliente Jersey con la API de MercadoLibre. El enunciado dice que tenemos que usarla para conseguir de los países, provincias y ciudades.
Con lo que dice el enunciado y el ejemplo del repo, planteamos el siguiente código, pero no tenemos idea que devuelve o que es el ClientResponse, osea no sabemos como manejarlo.

import com.sun.jersey.api.client.Client;
import com.sun.jersey.api.client.ClientResponse;
import com.sun.jersey.api.client.WebResource;

import javax.ws.rs.core.MediaType;

public class DireccionPostal {

    String calle;
    String altura;
    String piso;
    String departamento;
    ClientResponse cliente;

    void pais() {
        cliente =  Client.create()
           .resource("https://api.mercadolibre.com/%22)
           .path("classified_locations/countries")
           .accept(MediaType.APPLICATION_JSON) 
           .get(ClientResponse.class);

    }
}

**Comment by roli93** [comment]
@TomasM99 Buenas! Fijense en la docu :D http://jersey.java.net/documentation/latest/client.html

**Comment by roli93** [comment]
@TomasM99 Les sirvió la respuesta?

**Comment by TomasM99** [comment, marked as answer]
Buenas! Perdona la demora de la respuesta, es que estuvimos viendo la documentación estos días, y si gracias! Con eso y los ejemplos pudimos empezar a encararlo, cualquier duda que surja volvemos a preguntar mas puntualmente, saludos
